app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -25,6 +25,7 @@ def validate_service_provider_and_authn_context
   def add_sp_metadata_to_session
     session[:sp] = { loa3: loa3_requested?,
                      logo: current_sp_metadata[:logo],
+                     issuer: saml_request.service_provider.identifier,
```

Comment:
I duplicated this method a bit in `openid_connect/authorization_controller.rb` -- can you rebase and add this there too? 😀 I can try to refactor it after too so we can share more code